pinot-controller/src/test/java/org/apache/pinot/controller/recommender/TestConfigEngine.java

```diff
@@ -125,6 +125,18 @@ void testInvertedSortedIndexJointRule()
     assertEquals(output.getIndexConfig().getSortedColumn(), "c");
   }
 
+  @Test
```

Comment:
There is an existing test `testInvertedSortedIndexJointRule()`
Between that and this newly added test, can we ensure that recommendation is covered for dimensions, metrics, timeSpec (deprecated) and dateTimes. I think metric is not covered. Rest seems to be covered ?

Reply:
right, will update

Reply:
covered in same test case